### PR TITLE
Create css-has.json

### PR DESCRIPTION
**Pull request by yukulele**

### CODE REVIEW

**Comment by Fyrd** [comment]
@yukulele Your `bugs` array should just be `[]`, currently there's an error because it has an empty "bug" object in it.

File: features-json/css-has.json [hunk 1]
```diff
@@ -0,0 +1,256 @@
+{
+  "title":":has() CSS relational pseudo-class",
+  "description":"Only select elements containing specified content. for example, `a:has(>img)` select all `<a>` elements that contains a `<img>` child",
+  "spec":"https://drafts.csswg.org/selectors-4/#relational",
+  "status":"wd",
```
Comment (cvrebert) [inline features-json/css-has.json]: `:has()` is only in the ED, not [the WD](https://www.w3.org/TR/selectors4/), so this should be `"unoff"`

**Comment by cvrebert** [comment]
Would be good to include https://bugzilla.mozilla.org/show_bug.cgi?id=418039 in `links`

**Comment by yukulele** [comment]
* @Fyrd fixed, thanks!
* @cvrebert added, thanks!

**Comment by Fyrd** [comment]
Thanks! I should note though that ordinarily features that aren't yet being implemented anywhere aren't added to the site. I may make an exception to this one though since I know developers have been wanting this since forever.

**Pull request merged**